modules/plugin/coverage-multidim/netcdf/src/main/java/org/geotools/coverage/io/netcdf/NetCDFReader.java

```diff
@@ -142,6 +144,12 @@
 
     String defaultName = null;
 
+    static {
+        if (Boolean.getBoolean("org.geotools.coverage.io.netcdf.coordSysEnhanceOnly")) {
+            NetcdfDataset.setDefaultEnhanceMode(EnumSet.of(NetcdfDataset.Enhance.CoordSystems));
+        }
+    }
+
```

Comment:
Is this the right place where to put the systemProperty check?
I have seen other EnhancingMode in SPI and Utilities.